nixos/modules/system/boot/systemd.nix

```diff
@@ -406,20 +407,16 @@ in
       '';
     };
 
-    enableCgroupAccounting = mkOption {
-      default = true;
-      type = types.bool;
-      description = ''
-        Whether to enable cgroup accounting; see {manpage}`cgroups(7)`.
-      '';
-    };
-
-    extraConfig = mkOption {
-      default = "";
-      type = types.lines;
-      example = "DefaultLimitCORE=infinity";
+    settings.Manager = mkOption {
```

Comment:
I forget, is there a reason we can’t just set the defaults in `default` now? (Merging weirdness?)

This should probably have a `defaultText` showing the divergences we make from the stock configuration, in any case.

Reply:
> I forget, is there a reason we can’t just set the defaults in `default` now? (Merging weirdness?)

With `default = { Foo = "bar"; };`, if any user sets `settings.Manager = /* literally anything */;` it will overwrite the defaults entirely and remove them.

Reply:
Ah, yes. I forgot about that annoying detail.